templates/zerver/help/zulip-on-linux.md

```diff
@@ -0,0 +1,40 @@
+# Using Zulip on Linux
+
+Zulip on the Linux desktop is even better than Zulip on the web, with a cleaner look, native notifications, and support for multiple Zulip accounts.
+For download instructions, go to the [desktop app installation guide](https://zulipchat.com/help/desktop-app-install-guide#linux).
+
+
+## Initial setup
+
+### Connecting to a server
+
+When you open the Zulip desktop app for the first time, a screen asking
+for your server's or organization's address will appear. You have to write the server's URL in it, and
+press **Save**.
+
+!!! warn ""
+    **Note:** There's no need to add `https://` in case your server is running a secure
+    HTTP (HTTPS) protocol.
+
+    However, if your server uses an insecure HTTP connection, you'll have to
+    write `http://` manually before the server's URL. A message will appear, warning you about the risks
```

Comment:
I thought `https://` is enforced in a zulip server? With the certificate being either already existing, self-signed, or from certbot.

Reply:
This same thing is mentioned in the documentation for [*Zulip on Windows*](https://github.com/zulip/zulip/blob/master/templates/zerver/help/zulip-on-windows.md) and [*Zulip on Android*](https://github.com/zulip/zulip/blob/master/templates/zerver/help/zulip-on-windows.md)
 
What I thought is, Zulip servers have `https://` enforced in them, but what about organizations that are using their own servers, and may not be using HTTPS protocol?

Reply:
I suspect `http://` is primarily for interacting with a Zulip development server.

Reply:
Oh, so should I mention this warning message in the document?